### DISCUSSION TITLE
failed to start v2ray-core: unexpected exiting: check the log for more information****

### DISCUSSION THREAD

**Discussion by konglquan** [v2rayA/v2rayA#1466] [Q&A]
2024/08/05 20:17:55.172 [I] [process.go:101] low memory: 1563MiB, set V2RAY_CONF_GEOLOADER=memconservative
2024/08/05 20:17:55.199 [I] [io.go:431] V2Ray 5.16.1 (V2Fly, a community-driven edition of V2Ray.) Custom (go1.22.2 linux/amd64)
2024/08/05 20:17:55.199 [I] [io.go:431] A unified platform for anti-censorship.
2024/08/05 20:17:55.425 [I] [io.go:431] Failed to start: app/proxyman/inbound: failed to listen TCP on 20170 > transport/internet: failed to listen on address: [::1]:20170 > transport/internet/tcp: failed to listen TCP on [::1]:20170 > listen tcp [::1]:20170: bind: cannot assign requested address
2024/08/05 20:17:55.425 [I] [io.go:431]
2024/08/05 20:17:55.430 [W] [asm_amd64.s:1695] v2ray-core: exit status 1
2024/08/05 20:17:55.474 [E] [connection.go:88] failed to start v2ray-core: unexpected exiting: check the log for more information****

**Comment by gugu-hw** [comment]
I solved this issue by enableling ipv6. Check if you disabled ipv6 if not on purpose. If you must work on ipv4, then no help for now.

**Reply by NOX69RUS** [reply]
The method with enabling/disabling IPv6 didn’t help me. BUT, I solved the problem differently:

1. Disable V2rayA using any convenient method.
2. Log in to the router management console via SSH.
3. Delete the contents of the file geosite.dat. To do this, enter the command nano /usr/share/xray/geosite.dat in the console. Press the key combination Alt + T (deletes everything), then press Ctrl + X (exit nano), and confirm the changes by pressing Y (yes).
**OR**
rm /usr/share/xray/geosite.dat
touch /usr/share/xray/geosite.dat
5. Enable V2rayA and go to its settings screen at 192.xx.xx.xx:2017, select your VPN connection, and press the CONNECT button.
 
In my case, the connection started immediately.

**Reply by Y-Akamirsky** [reply]
@NOX69RUS Thanks! Your method helped me at 2025/2026 :) But i did this deletion easiest way - just press "Delete" button in GFWList update window.
